.vscode/settings.json

```diff
@@ -1,15 +1,14 @@
 {
-  "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
-  },
+  "editor.codeActionsOnSave": { "source.fixAll.eslint": true },
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
   "eslint.rules.customizations": [{ "rule": "*", "severity": "warn" }],
   "eslint.workingDirectories": [
     { "pattern": "apps/*/" },
     { "pattern": "packages/*/" }
   ],
-  "typescript.tsdk": "node_modules/typescript/lib",
+  "[prisma]": { "editor.defaultFormatter": "Prisma.prisma" },
```

Comment:
Technically this should go first if were doing alphabetical order

Reply:
Yeah, nice spot; hadn't thought of that.